vendor/.htaccess

```diff
@@ -1,10 +0,0 @@
-# Apache 2.2
```

Comment:
Careful I think you removed this file and commited the change by mistake, you need to revert this change and then it's good forme